### DISCUSSION TITLE
How to do calculations to find the abundances of each genome

### DISCUSSION THREAD

**Discussion by jackieway** [metagenome-atlas/atlas#716] [General]
Hello,

I'm looking at the relative abundances of my MAGs in each of my samples using the median_coverage_genomes.parquet file in the /genomes/counts folder. I was wondering how I would go about manually calculating the relative abundances of each of my genomes in my samples. I read in the ATLAS paper that the reads are mapped to the non-redundant MAGs to determine the median coverage across each genome, how exactly would I do this? I tried using a different binner on my assembled contigs to compare with the metagenome-atlas results, but am not sure how to determine relative abundance of these bins outside the metagenome-atlas workflow.

**Comment by SilasK** [comment]
Check out the Tutorial. There I have R and python code to calculate and analyze relative abundance. https://github.com/metagenome-atlas/Tutorial
